components/conversation/apiHelpers.js

```diff
@@ -1,13 +1,26 @@
 import conversationMockData from '../../data/conversationMockData.js';
 import data from '../../data/data.js';
+import axios from 'axios';
 
+const CHAT_API = 'http://3.133.100.147:2550';
```

Comment:
Ah smart making a variable that is much cleaner than just running the full string every time